File: conference/registration-bc/web/src/main/java/com/microsoft/conference/registration/commands/Orders/AssignRegistrantDetails.cs
using System;
using ENode.Commanding;

namespace Registration.Commands.Orders
{
    public class AssignRegistrantDetails : Command<Guid>
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }

        public AssignRegistrantDetails() { }
        public AssignRegistrantDetails(Guid orderId) : base(orderId) { }
    }
}

File: samples/conference/common/src/main/java/com/microsoft/conference/common/registration/commands/Orders/AssignRegistrantDetails.java
package com.microsoft.conference.common.registration.commands.Orders;

import org.enodeframework.commanding.Command;

public class AssignRegistrantDetails extends Command<String> {
    public String FirstName;
    public String LastName;
    public String Email;

    public AssignRegistrantDetails() {
    }

    public AssignRegistrantDetails(String orderId) {
        super(orderId);
    }
}
